src/components/action-list/ActionList.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const DIRECTION = {
+  toRight: 'to-right',
+  toTop: 'to-top',
+  centered: 'centered',
+  spaceBetween: 'space-between'
+};
+
+const ActionList = ({holes = [], direction, noWrap}) => {
+  const actionListClass = classNames('sg-actions-list', {
+    [`sg-actions-list--${direction}`]: direction,
+    'sg-actions-list--no-wrap': noWrap
+  });
+
+  return (
+    <div className={actionListClass}>
+      {holes.map((hole, i) => <div key={i} className="sg-actions-list__hole">{hole}</div>)}
```

Comment:
there are no modifiers for now.
i would stick to current solution and when we decide to add modifiers we can create separated component